File: MoviesExample/core/src/main/java/com/novoda/demo/movies/api/MoviesResponse.java
package com.novoda.demo.movies.api;

import com.novoda.demo.movies.model.Movie;

import java.util.List;

public class MoviesResponse {

    public int page;

    public List<Movie> results;

    public int total_pages;

    public int total_results;

    @Override
    public String toString() {
        return "MoviesResponse{" +
                "page=" + page +
                ", total_pages=" + total_pages +
                ", total_results=" + total_results +
                '}';
    }
}

File: MoviesExample/core/src/main/java/com/novoda/demo/movies/api/MoviesResponse.kt
package com.novoda.demo.movies.api

import com.novoda.demo.movies.model.Movie

data class MoviesResponse(
        var page: Int = 0,
        var results: List<Movie> = emptyList(),
        var total_pages: Int = 0,
        var total_results: Int = 0
)
